charts/stellar-network/values.yaml

```diff
@@ -29,3 +33,200 @@ stellarNetwork:
     # labels:
     #   environment: "dev"
     #   mylabel1: "myvalue1"
+
+# Reynholm Industries Network Validator definition
```

Comment:
We should use defaults and not publish the demo network's config for now. Especially with secrets.